sdk/instrumentation/google.golang.org/grpc/server.go

```diff
@@ -96,7 +96,11 @@ func wrapHandler(
 			setTruncatedBodyAttribute("request", reqBody, int(dataCaptureConfig.BodyMaxSizeBytes.Value), span)
 
 			if md, ok := metadata.FromIncomingContext(ctx); ok {
-				if filter.EvaluateBody(span, reqBody, md) {
+				var processingBody []byte
+				if int(dataCaptureConfig.BodyMaxProcessingSizeBytes.Value) > len(reqBody) {
```

Comment:
`reqBody` is the decoded value of the body right?  In which case, we should just use the same logic as https://github.com/hypertrace/goagent/pull/160/files#diff-c9de109efb7aa8920966082d6c5481c3324292b143637ff5c5f3f2174a5e1b9dR93